### ISSUE TITLE
Add button to view dipole vector

### ISSUE THREAD

**Issue by ghutchis** [pittquantum/PittQuantumRepository#88]
Next to the dipole moment property, there should be a small button to toggle viewing the dipole moment vector:

glviewer.addArrow({end: new $3Dmol.Vector3(1.0, 2.0, 3.0), color: "black", wireframe: false});

The components are in pm7: dipole  (the pm7 will change with the calculation method)

"dipole": [
      "-0.001", 
      "-0.001", 
      "-0.074"
    ],

**Comment by RitwikGupta** [comment]
See 9c699e1. This is currently a work in progress as soon as @vonbearshark finishes the new JS build.

**Comment by RitwikGupta** [comment]
978199a6440971a6bc49b278583a12b57ac733ee adds the dipole vector on document load. We can make this a button.

**Comment by ghutchis** [comment]
I don't think we want this to show by default. For some molecules, this will look cluttered.

**Comment by RitwikGupta** [comment]
No worries, this is temporary till we add a button
